bin/auth-api/src/routes/auth.routes.ts

```diff
@@ -74,7 +74,11 @@ router.post("/auth/login", async (ctx) => {
     throw new ApiError("Forbidden", "You do not have access to that workspace");
   }
 
-  const token = createSdfAuthToken(user.id, workspaceId);
+  const token = createSdfAuthToken({
+    userId: user.id,
+    workspaceId,
+    role: "web",
```

Comment:
do we really want to hard code a role of "web" - can't this at least be an enum we extend?

Reply:
The enum is here: https://github.com/systeminit/si/pull/5143/files#diff-45a5a885b573adda0e5349e1af441abf982f4f977141755f48dd5780d52a17bdR52

Basically in TS (at least in my experience) unless an enum has extra data they are generally a type union of the possible values, and you specify those values directly. TS will yell at you if you don't use one of them.

Reply:
TS is being weird about the enum, but the line is in auth.service.ts:

```typescript
export type SdfAuthTokenRole = "web" | "automation";
```

And it's part of the type on the parameter you pass to `createSdfAuthToken`, which is how TS knows what values are allowed.